### PR TITLE
Final style

### PR DESCRIPTION
**Pull request by Johnny8607**
Errors in the reservation form page that needs to be fixed (no biggie, the form still works).
Accept the pr after Sevag's one.

### CODE REVIEW

File: Sprint 4/soen341-rentalapp/app/page.tsx [hunk 1]
```diff
@@ -1,155 +1,147 @@
...
         ))}
+      </div> */}
+
+      <div className="flex flex-row justify-center items-center gap-56 mt-28 px-20">
+        <div className="flex items-center">
+          <div className="flex flex-col gap-10 max-w-3xl bg-blue-100 p-14 rounded-lg">
+            <h1 className="font-bold text-7xl">Welcome To <span className={`mr-1 text-7xl`}>Rental</span><span className={`${kanit.className} text-8xl font-bold text-blue-800`}>341</span></h1>
+            <p className=" text-xl text-gray-500">Discover seamless travel. Choose from our diverse fleet, book effortlessly, and hit the road with confidence. Your adventure awaits!</p>
+            <div className="flex flex-row gap-10">
+              <a href="/register"><button className="flex flex-row gap-2 items-center font-bold text-lg text-slate-200 w-max rounded-md bg-blue-700 hover:bg-blue-600 px-6 py-3">Book Ride <FaRegCheckCircle /></button></a>
```
Comment (egadb) [inline Sprint 4/soen341-rentalapp/app/page.tsx:41]: use nextjs Link component isntead of button/a component

File: Sprint 4/soen341-rentalapp/app/page.tsx [hunk 2]
```diff
@@ -1,155 +1,147 @@
...
+              src={confirmIcon} 
+              alt="Confirm Icon"
+              className="w-20"
+              />
+            <h1 className="text-4xl font-bold">Confirm</h1>
+            <p className="text-gray-500">Finalize your vehicle choice now to secure your ride for the journey ahead. With a simple confirmation, your adventure awaits. Let's make it happen!</p>
+          </div>
+        </div>
+        <div className="mt-12">
+        <a href="/register"><button className="flex flex-row items-center px-12 font-bold text-xl bg-blue-700 hover:bg-blue-600 text-slate-100 rounded-lg"> I'm Ready! <Lottie className=" max-w-20" animationData={animationData2}/></button></a>
```
Comment (egadb) [inline Sprint 4/soen341-rentalapp/app/page.tsx:82]: same here

File: Sprint 4/soen341-rentalapp/app/page.tsx [hunk 3]
```diff
@@ -1,155 +1,147 @@
-import React from "react";
+'use client'
```
Comment (egadb) [inline Sprint 4/soen341-rentalapp/app/page.tsx:1]: dont need use client here unless needed for infinitecard thing

File: Sprint 4/soen341-rentalapp/next.config.mjs [hunk 4]
```diff
@@ -5,6 +5,11 @@ const nextConfig = {
       {
         protocol: "https",
         hostname: "www.motortrend.com",
+        hostname: "purepng.com",
+        hostname: "p7.hiclipart.com",
+        hostname: "pngimg.com",
+        hostname: "www.pngall.com",
+        hostname: "www.motortrend.com"
```
Comment (egadb) [inline Sprint 4/soen341-rentalapp/next.config.mjs:12]: duplicated motortrend

File: Sprint 4/soen341-rentalapp/app/page.tsx (hunk 3 above)
Comment (Johnny8607) [reply Sprint 4/soen341-rentalapp/app/page.tsx:1]: I've tried removing it but it doesnt work bc one of the library uses a hook :D

**Review by SevagMerdkhanian — COMMENTED** [review]
Please fix merge conflicts in NavBar.tsx :)

**Review by SevagMerdkhanian — APPROVED** [review]

**Pull request merged**